Kitodo/src/main/java/org/kitodo/config/enums/ParameterCore.java

```diff
@@ -367,7 +367,7 @@ public enum ParameterCore implements ParameterInterface {
     /**
      * Validation of process title via regular expression.
      */
-    VALIDATE_PROCESS_TITLE_REGEX(new Parameter<>("validateProzessTitelRegex", "[\\w-]*")),
+    VALIDATE_PROCESS_TITLE_REGEX(new Parameter<>("validateProzessTitelRegex", "^[a-zA-Z0-9_-]+$")),
```

Comment:
Should the new default value not even updated in the `kitodo_config.properties` files?

Reply:
I am wondering: This is a hard limitation of the Kitodo software, some titles lead to problems later. Should we allow to change that in the config.properties at all? Or should we hardcode that in the application? Additionally one can think of allowing **additional** regexes which might be checked on top of the minimal requirements. See https://github.com/kitodo/kitodo-production/issues/6055

cc @solth

Reply:
At SLUB we allow even the `.` inside the process title as for historical reasons this was needed to create the processes. Maybe this can be changed now but this should be discussed.

Reply:
One more note: the `w` meta character in regular expressions stands for `a-z, A-Z, 0-9, including _ (underscore)` . So the expression can be simplified to `^[\\w-]+$`?

Reply:
What about chosing a new parameter name "validateProcessTitle" which avoids the current "denglisch" (English-German mix)?

Reply:
> One more note: the `w` meta character in regular expressions stands for `a-z, A-Z, 0-9, including _ (underscore)` . So the expression can be simplified to `^[\\w-]+$`?

Thanks i changed that. It turned out that `[\\w-]+` in combination with Java's `matches` als works, but i think the changed regex is more expressive.  

>What about chosing a new parameter name "validateProcessTitle" which avoids the current "denglisch" (English-German mix)?

I also have sympathy for that, but that would require institutions having to adapt their config. I think the Pull Request should probably be backported which maybe would probably be easier, when we do not change configuration.